docs/source/examples/workspaces/azure/PinFile

```diff
@@ -125,3 +125,18 @@ azure_vm_test:
         credentials:
           filename: azure.key
           profile: key
+
+azure_api:
+  topology:
+    topology_name: azure_api
+    resource_groups:
+      - resource_group_name: "azure_api"
+        resource_group_type: "azure"
+        resource_definitions:
+          - role: "azure_api"
+            resource_group: ccit
+            url: "https://management.azure.com/subscriptions/dcc74c29-4db6-4c49-9a0f-ac0ee03fa17e/resourceGroups/ccit/providers/Microsoft.Compute/virtualMachines/test?api-version=2019-03-01"
```

Comment:
I prefer any specific information such as our subscription will not be in the examples

Reply:
OK will take this away